catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -148,6 +153,7 @@ protected Object executeWithSubject() throws Exception {
                     } catch (IOException | CatalogTransformerException e) {
                         LOGGER.debug("Could not transform metacard: {}", id);
                     }
+                    metacard = applyInjectors(metacard, attributeInjectors);
```

Comment:
❓ The catalog framework should inject attributes.  Is there a reason the command needs to duplicate that work?